ui/StatusQ/src/StatusQ/Controls/StatusTagItem.qml

```diff
@@ -26,30 +26,30 @@ import StatusQ.Core.Theme 0.1
 
      For a list of components available see StatusQ.
   */
-Rectangle {
+Control {
     id: root
 
     /*!
-        \qmlproperty bool StatusTagItem::isReadonly
-        This property sets if the tag is readonly or not.
-    */
+    \qmlproperty bool StatusTagItem::isReadonly
+    This property sets if the tag is readonly or not.
+*/
     property bool isReadonly
     /*!
-        \qmlproperty string StatusTagItem::name
-        This property sets the tag text to display.
-    */
+    \qmlproperty string StatusTagItem::name
+    This property sets the tag text to display.
+*/
     property string text
     /*!
-        \qmlproperty string StatusTagItem::icon
-        This property sets the tag icon to display.
-    */
+    \qmlproperty string StatusTagItem::icon
+    This property sets the tag icon to display.
+*/
     property string icon
 
     /*!
-        \qmlsignal
-        This signal is emitted when the tag is clicked.
-    */
-    signal clicked()
+    \qmlsignal
+    This signal is emitted when the remove user button is clicked.
+*/
+    signal removeUser()
```

Comment:
WDYT of keeping it generic since the `StatusTagItem` doesn't need to know anything about users. Something like `closeClicked` or `removeRequested` would work IMO.

Reply:
updated, even thought is not "closing" anything in reality but removing